libs/langchain-community/src/vectorstores/tests/upstash.int.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Index } from "@upstash/vector";
+import { OpenAIEmbeddings } from "@langchain/openai";
+import { Document } from "@langchain/core/documents";
+import { UpstashVectorStore } from "../upstash.js";
+import { sleep } from "../../utils/time.js";
+
+describe("UpstashVectorStore", () => {
+  let store: UpstashVectorStore;
+  let embeddings: OpenAIEmbeddings;
+  let index: Index;
+
+  beforeEach(async () => {
+    index = new Index({
+      url: process.env.UPSTASH_VECTOR_REST_URL,
+      token: process.env.UPSTASH_VECTOR_REST_TOKEN,
+    });
+
+    embeddings = new OpenAIEmbeddings({
+      openAIApiKey: process.env.OPENAI_KEY,
+    });
```

Comment:
```suggestion
    embeddings = new SyntheticEmbeddings({
      vectorSize: 1536
    });
```
Import from `import { SyntheticEmbeddings } from "@langchain/core/utils/testing";`